python/cudf/cudf/_lib/nvtext/subword_tokenize.pyx

```diff
@@ -36,8 +36,7 @@ def subword_tokenize_inmem_hash(
     uint32_t max_sequence_length=64,
     uint32_t stride=48,
     bool do_lower=True,
-    bool do_truncate=False,
-    uint32_t max_rows_tensor=500
+    bool do_truncate=False
```

Comment:
You can leave trailing commas on the last parameter. In cases like this, it has no effect on Python parsing but does reduce the diff from two lines to one.
```suggestion
    bool do_truncate=False,
```